templates/ilvo/ILVO Sample plant.json

```diff
@@ -0,0 +1,114 @@
+{
+    "metadata": {
+      "name": "sample_collection",
+      "group": "arrayexpress",
+      "group_order": 2,
+      "temporary_name": "2_arrayexpress_sample_collection",
+      "template_version": "1.2.0",
+      "isa_config": "studySample_default_v2015-07-02",
+      "isa_measurement_type": "Sample",
+      "isa_technology_type": null,
+      "isa_protocol_type": "sample collection",
+      "repo_schema_id": "Annotare-default.properties",
+      "organism": "any",
+      "level": "study"
+    },
+    "data":[
+    {
+      "iri": null,
+      "name": "Input",
+      "description": "Existing Samples in DataHub that represent input for this protocol.",
+      "dataType": "SEEK Sample Multi",
+      "required": true,
+      "isaTag": null
+    },
+    {
+      "iri": "http://www.ebi.ac.uk/efo/EFO_0005518",
+      "name": "sample collection",
+      "description": "A protocol that describes the procedure whereby biological samples for an experiment are sourced. Examples of this are: collection of clinical samples from patients, collection of plant materials in a forest, collection of larvae from lab-grown Drosophila flies.",
+      "dataType": "String",
+      "required": true,
+      "isaTag": "protocol",
+      "ontology": null,
+      "CVList": null
+    }, 
+    {
+      "iri": "http://www.ebi.ac.uk/efo/EFO_0000688",
+      "name": "sampling site",
+      "description": "A site from which a sample, i.e. a statistically representative of the whole, is extracted from the whole. e.g. a liver sample.",
+      "dataType": "String",
+      "required": false,
+      "isaTag": "parameter_value"
+    },
+    {
+        "iri": "",
```

Comment:
Might be best to have this iri as `null` and not empty

Reply:
ok